docs/modules/customize/pages/index.adoc

```diff
@@ -16,6 +16,7 @@ Inside of your application you can customize it in different ways:
 
 * xref:customize:authorizations.adoc[Authorizations]
 * xref:customize:code.adoc[Code]
+* xref:customize:data_consent.adoc[Data consent]
 * xref:customize:gemfile.adoc[Gemfile]
 * xref:customize:images.adoc[Images]
 * xref:customize:javascript.adoc[Javascript]
```

Comment:
I see that we're also missing the [Logic page](https://docs.decidim.org/en/customize/logic.html). Can you add it 🙏🏽? 
```suggestion
* xref:customize:javascript.adoc[Javascript]
* xref:customize:logic.adoc[Logic]
```